cli/src/pcluster/config/validators.py

```diff
@@ -1539,3 +1546,31 @@ def duplicate_shared_dir_validator(section_key, section_label, pcluster_config):
                 errors.append("'shared_dir' can not be specified in cluster section when using multiple EBS volumes")
 
     return errors, warnings
+
+
+def ebs_volume_throughput_validator(section_key, section_label, pcluster_config):
+    errors = []
+    warnings = []
+
+    section = pcluster_config.get_section(section_key, section_label)
+    volume_type = section.get_param_value("volume_type")
+    volume_iops = section.get_param_value("volume_iops")
+    volume_throughput = section.get_param_value("volume_throughput")
+
+    if volume_type == "gp3":
+        min_throughput, max_throughput = 125, 1000
+        if volume_throughput < min_throughput or volume_throughput > max_throughput:
+            errors.append(
+                "Throughput must be between {min_throughput} MB/s and {max_throughput} MB/s when provisioning "
+                "{volume_type} volumes.".format(
+                    min_throughput=min_throughput, max_throughput=max_throughput, volume_type=volume_type
+                )
+            )
+        if volume_throughput and volume_throughput > volume_iops * 0.25:
```

Comment:
can we assign 0.25 to a variable in order to better understand what this magic number refers to?

Reply:
Done!